src/main/java/com/redhat/syseng/soleng/rhpam/processmigration/model/Execution.java

```diff
@@ -26,11 +21,17 @@
 
     private ExecutionType type;
 
-    @JsonProperty("callback_url")
```

Comment:
Why remove the `@JsonProperty` annotations? they are useful for formatting the request/response objects

Reply:
added back